recipes/configs/llama3/70B_full.yaml

```
@@ -86,17 +86,18 @@ optimizer:
 loss:
   _component_: torchtune.modules.loss.CEWithChunkedOutputLoss
 max_steps_per_epoch: null
-gradient_accumulation_steps: 1
+gradient_accumulation_steps: 1  # Use to increase virtual batch size
 
 # Training env
 device: cuda
 
 # Memory management
-enable_activation_checkpointing: True
+enable_activation_checkpointing: True  # True reduces memory
 enable_activation_offloading: False  # True reduces memory
-custom_sharded_layers: ['tok_embeddings', 'output']
+custom_sharded_layers: ['tok_embeddings', 'output']  # Shard it separetely. Useful for large vocab size. Lower Memory, but lower speed
```

So we did decide to keep this around? Also I feel like we are pushing the limits of what we can effectively convey in an inline comment with this one
```suggestion
custom_sharded_layers: ['tok_embeddings', 'output']  # Layers to shard separately (useful for large vocab size models). Lower Memory, but lower speed
```